packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -88,19 +130,42 @@ class Send extends Component {
                     onSubmit={this.handleSubmit}
                     validate={this.validateForm}
                     decorators={[this.decorator]}
-                    render={({ handleSubmit, valid, validating, values }) => (
+                    mutators={{ recalculateMax }}
+                    render={({
+                      handleSubmit,
+                      valid,
+                      validating,
+                      values,
+                      form: { mutators }
+                    }) => (
                       <form className='send-form' onSubmit={handleSubmit}>
                         <fieldset className='form_fields'>
                           <Field
                             className='form_field_amount'
                             formNoValidate
                             label='Amount'
                             name='amount'
+                            disabled={this.state.maxSelected}
                             placeholder='0.00'
                             render={FetherForm.Field}
                             required
                             type='number' // In ETH or coin
-                          />
+                          >
+                            <button
+                              type='button'
+                              className={
+                                this.state.maxSelected
+                                  ? 'button -tiny active max'
+                                  : 'button -tiny max'
+                              }
+                              onClick={() => {
```

Comment:
create a class method called `handleMaxClick`, and here put `onClick={this.handleMaxClick}`

Reply:
I can't unfortunately because of the mutators